src/main/java/eu/stamp_project/testrunner/runner/coverage/JacocoRunner.java

```diff
@@ -127,6 +144,16 @@ public Coverage run(String classesDirectory,
     protected abstract Coverage executeTest(String[] testClassNames,
                                             String[] testMethodNames,
                                             List<String> blackList);
+    
+    
+    
+    
+    protected Coverage executeTest(String[] testClassNames,
```

Comment:
coverage seems to be the output object, what information does it contain?

Reply:
Coverage contains, for example, a method that returns the number of covered lines, but, at the same time, it declares a method for collecting data. Due to that method, I passed as input.